build-system/pr-check.js

```diff
@@ -207,6 +207,12 @@ function main(argv) {
     return 0;
   }
 
+  if (files.includes('package.json') ? !files.includes('yarn.lock') : false) {
```

Comment:
I think we I suggested previously was correct:

if has package.json => must also have yarn.lock, otherwise must NOT have yarn.lock